utils/dateUtils.ts

```diff
@@ -188,18 +188,44 @@ export function formatTimestampSimple(date: Date): string {
   return format(date, 'PPpp')
 }
 
+// Function to check if the date is today (from midnight to now)
 export function isToday(date: Date): boolean {
-  return isTodayDate(date)
+  const now = new Date();
+  return isWithinInterval(date, {
+    start: startOfDay(now),
+    end: now, // Up to the current time
+  });
 }
 
+// Function to check if the date is within the past 7 days, including last 24 hours but excluding today
 export function isPast7Days(date: Date): boolean {
-  return isWithinInterval(date, { start: subDays(new Date(), 7), end: new Date() })
+  const now = new Date()
+  const sevenDaysAgo = subDays(now, 7)
+  const twentyFourHoursAgo = subHours(now, 24)
+  
+  return (
+    isWithinInterval(date, { 
+      start: sevenDaysAgo, 
+      end: now 
+    }) &&
+    !isToday(date) &&
+    (date >= twentyFourHoursAgo || date < startOfDay(now))
```

Comment:
Is this last line necessary? It's a little confusing the me because you've already filtered out today's conversations.

`date >= twentyFourHoursAgo` technically isn't correct because we wouldn't want to include convos from today (but again those are already filtered out).

`date < startOfDay(now)` is also already handled with the ` !isToday(date)` condition so I don't think this line actually filters anything out

Reply:
should be changed! thank you